app/mailers/conversation_reply_mailer.rb

```diff
@@ -115,6 +115,14 @@ def from_email_with_name
     end
   end
 
+  def channel_email_with_name
+    if @conversation.assignee.present?
+      I18n.t('conversations.reply.channel_email.header.reply_with_name', assignee_name: assignee_name, from_email: @channel.email)
```

Comment:
Let us use the same format "Pranav from Chatwoot" instead of "Pranav via hello@chatwoot.com".

Reply:
Okay 👍🏼